ecosystem/typescript/sdk/src/util.test.ts

```diff
@@ -0,0 +1,6 @@
+export const NODE_URL = process.env.APTOS_NODE_URL || "https://fullnode.devnet.aptoslabs.com";
+export const FAUCET_URL = process.env.APTOS_FAUCET_URL || "https://faucet.devnet.aptoslabs.com";
+
+test("noop", () => {
+  // This is just a hack to have a non-compiled test util :-)
```

Comment:
I don't know what this means or why, could you elaborate so that someone doesn't delete it?